src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_menu">
+      <a href="https://www.moyo.ua/" class="menu_logo">
+        <img
+        src="images/logo.png"
+        alt="moyo logo"
+        class="image_logo">
```

Comment:
check the code formatting (checklist - point 13)
![image](https://github.com/antstania/layout_moyo-header/assets/22824431/0b3b3ecc-ecd8-4d05-87f6-fba188ef0b99)